src/simsopt/util/coil_util.py

```diff
@@ -0,0 +1,72 @@
+def importCurves(file):
```

Comment:
How is this different from the load_curves_from_file method? Can we distinguish them in an easy way?

Reply:
I added these by mistake on my last commit. I will remove them right away. I used them while I was still testing load_curves_from_file.